packages/react/src/components/Modal/Modal.mdx

```diff
@@ -96,11 +96,18 @@ const ModalStateManager = ({
   );
 };
 ```
+
 <Unstyled>
-  <InlineNotification title="Warning" kind="warning" className="sb-notification">
-    <CodeSnippet type="inline" hideCopyButton>{'<Modal>'}</CodeSnippet>/<CodeSnippet type="inline" hideCopyButton>{`<ComposedModal>`}</CodeSnippet> has to be put at the
-    top level in a React tree. The easiest way to ensure that is using React
-    portal as shown in above example.
+  <InlineNotification
+    title="Warning"
+    kind="warning"
+    className="sb-notification">
+    <CodeSnippet type="inline" hideCopyButton>
+      {'<Modal>'}
+    </CodeSnippet>
+    /<CodeSnippet type="inline" hideCopyButton>{`<ComposedModal>`}</CodeSnippet>{' '}
+    has to be put at the top level in a React tree. The easiest way to ensure
+    that is using React portal as shown in above example.
```

Comment:
These `CodeSnippet` blocks seems to be causing crashes here and in `ComposedModal`

Reply:
Thanks for pointing this out. It should be fixed now.

To find valid syntax for .mdx files w/ MDX2, I've been doing the following:
1. Take the erroring portions (like this snippet here) and put it into the [mdx playground](https://mdxjs.com/playground/)
2. Starting at the most-nested components, reduce these down to single-line entries
```diff
- <CodeSnippet type="inline" hideCopyButton>
-  {'<Modal>'}
- </CodeSnippet>
+ <CodeSnippet type="inline" hideCopyButton>{'<Modal>'}</CodeSnippet>
```
3. Continue simplifying the formatting until the compiler is successful and doesn't report errors
4. Put the valid code over into the .mdx file and surround it with prettier ignore comments. These look a bit different in mdx2 and are surrounded in curly braces: `{/* <!-- prettier-ignore-start --> */}`